examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1179 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    bl = auto_model.layers[0].layers[0].transformer.layer[1]
</pre><p>Please add a sentence or two explaining which layer we're explaining and how to access/find it using from the transformer model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I added  'The auto model consists of a main BERT layer (layer 0) followed by two dense layers.&nbsp;</p><p>We extract the first transformer's block in the main BERT layer.'</p><p> I honestly don't know how to explain it better. </p>